src/gui_w32.c

```diff
@@ -7391,7 +7391,7 @@ add_dialog_element(
     *p++ = 0;  // advance pointer over nExtraStuff WORD   - 2 more
 
     return p;	//total = 15+ (strlen(caption)) words
-		//	   = 30 + 2(strlen(caption) bytes reqd
+		//	   = 30 + 2(strlen(caption) bytes read
```

Comment:
There is parenthesis mismatch on this line.
I did not fix it because I do not understand the comment.